From 4402cc589938c4b85ec41b522770f17e2012a1a6 Mon Sep 17 00:00:00 2001
From: Morris Hafner <morris.hafner@codeplay.com>
Date: Fri, 12 Jul 2019 15:27:35 +0100
Subject: [PATCH 3/3] Set CMAKE_C_STANDARD

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index adf6cb6..14e8db8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,8 +1,11 @@
-cmake_minimum_required (VERSION 2.8.11)
+cmake_minimum_required (VERSION 3.1)
 
 project (OPENCL_ICD_LOADER)
 find_package (Threads REQUIRED)
 
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_C_STANDARD_REQUIRED ON)
+
 # The option below allows building the ICD Loader library as a shared library
 # (ON, default) or a static library (OFF).
 #
-- 
2.17.1

